### PR TITLE
Address identicon handler

### PR DESCRIPTION
**Pull request by repoman**
Takes `GET` requests at `/identicon/<address>`. Returns string containing a 24 by 24 px div with the identicon as an inline background image.

### CODE REVIEW

File: route/identicon/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+router.get('/:address', function (req, res) {
+  const address = req.params.address;
+  const identiconDataUrl = identiconCanvas(address).toDataURL()
+
+  let el = `<div style="` +
+           `background-image: url('${identiconDataUrl}');` +
+           `background-size: 100%;` +
+           `width: 24px;` +
+           `height: 24px` +
+           `"></div>`;
```
Comment (nya-elimu) [inline route/identicon/index.js]: > Takes `GET` requests at `/identicon/<address>`. Returns string containing a 24 by 24 px div with the identicon as an inline background image.

@repoman Can we change the response type from `text/html` to `image/png` and then only return the `${identiconDataUrl}` in the response? (I.e. without any surrounding HTML code).

Screenshot 2020-07-20 at 14 19 42

File: app.js [hunk 2]
```diff
@@ -76,6 +76,10 @@ server.get('/votes', async (req, res) => {
   res.send()
 })
 
+const identiconRoute = require('./route/identicon/index');
+
```
Comment (nya-elimu) [inline app.js]: Remove the space between these two lines for readability?

**Review by nya-elimu — DISMISSED** [review]

**Review by jo-elimu — DISMISSED** [review]

**Comment by nya-elimu** [comment]
@repoman Here is the code that will be fetching the identicon: https://github.com/elimu-ai/webapp/pull/994/files#diff-eae9172dbb0a6920c34a4a010cbc7d97R56

Screenshot 2020-07-20 at 14 54 42

You can test it live here: http://hin.test.elimu.ai/contributions/aragon-dao

**Review by nya-elimu — DISMISSED** [review]

**Review by nya-elimu — APPROVED** [review]
Screenshot 2020-07-20 at 20 49 32

**Comment by nya-elimu** [comment]
Ready to be merged.

**Pull request merged**